cmd/uhc/urls/url_expander_test.go

```diff
@@ -0,0 +1,152 @@
+package urls
```

Comment:
Add the copyright header.

Reply:
Can we use ginkgo/gomega for this test?

Reply:
header added.

tests switched to gomega expectations.